db-service/test/cqn4sql/where-exists.test.js

```diff
@@ -1240,6 +1240,14 @@ describe('Path expressions in from combined with `exists` predicate', () => {
           AND EXISTS ( SELECT 1 from bookshop.Genres as parent where parent.ID = genre.parent_ID )
       `)
   })
+
+  it('MUST ... two EXISTS both on same path in where', () => {
+    let query = cqn4sql(CQL`SELECT from bookshop.Books { ID } where exists genre.children[code = 'ABC'] or genre.children[code = 'DEF']`, model)
```

Comment:
If you want to express to parallel exists you gotta write it like that
```suggestion
    let query = cqn4sql(CQL`SELECT from bookshop.Books { ID } where exists genre.children[code = 'ABC'] or exists genre.children[code = 'DEF']`, model)
```

Reply:
and then the correct, resulting cqn would be:

```
SELECT from bookshop.Books as Books { Books.ID }
          WHERE EXISTS (
            SELECT 1 from bookshop.Genres as genre where genre.ID = Books.genre_ID
              and EXISTS ( SELECT 1 from bookshop.Genres as children where children.parent_ID = genre.ID and children.code = 'ABC' )
          )
          or  EXISTS (
            SELECT 1 from bookshop.Genres as genre2 where genre2.ID = Books.genre_ID 
            and EXISTS ( SELECT 1 from bookshop.Genres as children2 where children2.parent_ID = genre2.ID and children2.code = 'DEF' )
          )
        `)

```